micrometer-core/src/main/java/io/micrometer/core/instrument/placeholder/RemovePlaceholdersNamingConvention.java

```diff
@@ -0,0 +1,38 @@
+package io.micrometer.core.instrument.placeholder;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.micrometer.core.instrument.util.StringUtils;
+import io.micrometer.core.lang.Nullable;
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * A naming convention that removes all placeholders from the metric name. Configured in
+ * non-hierarchical registries when using the placeholders mechanism. Metrics
+ * registered in non-hierarchical registries don't have tags encoded in their name, because
+ * they support tags explicitly.
+ *
+ * @author Piotr Betkier
+ */
+class RemovePlaceholdersNamingConvention implements NamingConvention {
```

Comment:
Not overwriting `getTag` and `getValue` here will effectively disable the tag processing for the delegate naming convention.